libraries/lib-module-manager/PluginManager.cpp

```diff
@@ -964,93 +992,6 @@ void PluginManager::SaveGroup(FileConfig *pRegistry, PluginType type)
    return;
 }
 
-// If bFast is true, do not do a full check.  Just check the ones
-// that are quick to check.  Currently (Feb 2017) just Nyquist
-// and built-ins.
-void PluginManager::CheckForUpdates(bool bFast)
```

Comment:
If the definition of `CheckPluginUpdates` is moved here, it makes the commit easier to review.  More black lines, fewer red and green.